docs/modules/ROOT/pages/access.adoc

```diff
@@ -106,6 +107,134 @@ after an initial stage with centralized administration is over.
 WARNING: Removing the owner altogether will mean that administrative tasks that are protected by `assert_only_owner`
 will no longer be callable!
 
+== `TwoStepOwnable`
```

Comment:
I think we don't need this section on this page, but rather maybe a note saying that a two-step mechanism for transferring the ownership can be implemented by using the same component, with a link to the API Reference.